### PR TITLE
Mcleodsn code review

### PR DESCRIPTION
**Pull request by mcleodsn4**
My three specific technical suggestions are:
1. Move the Styling to a separate CSS file. With using Thymeleaf we had to create a static folder under resources and put it in there. I have created the path and file and moved over the majority of you styling in there.

2. Create an endpoint for the "/" default endpoint. This allows for cleaner code making sure that the default endpoint is hit if nothing is entered.

3. With the dropdown for days it appeared that Wednesday-Saturday all had the same value. I changed these accordingly to make sure that each one had its unique value. This will most likely help you later on when needing to know what day certain events are on.

Additional Analysis
1. With this first sprint it seems like you may have not had enough stories tasked out. Maybe try doing a couple more stories this next sprint. Only having the base HTML seems to be a little light for this first sprint. Maybe try thinking of a different color scheme, the light blue on the black isn't the most appealing.
2. This project was available on Github on time.
3. There are no comments, but there really isn't that much code to comment/document out.
4. This program compiles and works fine in my browser.

### CODE REVIEW

**Comment by mcleodsn4** [comment]
Please Disregard the delete index.html, somehow when I pushed it created a second one that I deleted.

https://github.com/karliewinhoven/FeelFreeCincinnati/pull/1/commits/1ec7b9e62ff567b7f205392904c63e184510662e

**Comment by mcleodsn4** [comment]
Three Technical concepts that I learned 

1. I had not seen the integrity or cross-origin attributes on Bootstrap before. It is pretty interesting that the browser will not let the code be loaded if the source had been manipulated.

`integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous"`

2. I learned from your HTML that class="text-center" could be used to center the text. I had used this as a styling attribute, but never as a class.

`<div class="text-center">`

3. I learned that the &copy; created the copyright symbol. I had never had to use that before. I'm sure it will be helpful in the future!
`<footer>Copyright &copy; 2019 FeelFreeInc</footer>`

File: src/main/java/com/feelfreecincinncati/FeelFreeCincinnatiController.java [hunk 1]
```diff
@@ -12,4 +13,9 @@ public String start() {
 		return "index";
 	}
 
+	@RequestMapping("/")
+	public String error() {
```
Comment (discospiff) [inline src/main/java/com/feelfreecincinncati/FeelFreeCincinnatiController.java:17]: I'd... maybe call this method index()

File: src/main/resources/templates/index.html [hunk 2]
```diff
@@ -1,16 +1,11 @@
...
-<link href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+<link type="text/css" th:href="@{https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css}" rel="stylesheet" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
 <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
-<style>
-header {color: #00FFFF; background-color: #000000;}
-h1 {font-size: 50px}
-body {background-color: #BBBBBB}
-footer {text-align: center; font-size:75%; font-style: italic}
-</style>
+<link rel="stylesheet" type="text/css" th:href="@{/css/style.css}"/>
```
Comment (discospiff) [inline src/main/resources/templates/index.html:8]: Linked stylesheet is a good idea... better than inline, because you can use it across multiple pages.

**Comment by discospiff** [comment]
Several good changes here that reduce technical debt and move the program closer to commonly accepted standards.  I recommend merging this bran

**Pull request merged**